src/component/marker/MarkLineView.ts

```diff
@@ -190,9 +192,22 @@ function markLineFilter(
         ) {
             return true;
         }
+        const lineStart = coordSys.dataToPoint(item[0].coord);
+        const lineEnd = coordSys.dataToPoint(item[1].coord);
+        const area = (coordSys as Cartesian2D).getArea();
+        const isIntersect = linePolygonIntersect(
+            lineStart[0], lineStart[1], lineEnd[0], lineEnd[1],
+            [
+                [area.x, area.y], [area.width + area.x, area.y],
+                [area.x, area.height + area.y], [area.width + area.x, area.height + area.y]
```

Comment:
This seems to be shaped as "Z". Should the `[area.width + area.x, area.height + area.y]` be before `[area.x, area.height + area.y]`?